### DISCUSSION TITLE
Automatic snapshotting?

### DISCUSSION THREAD

**Discussion by jtagcat** [archlinux/archinstall#348] [Ideas]
- Periodic snapshots
 - Hooked snapshots (pre-pacman operation, post-pacman operation, etc)
 - Event snapshots (reboot/boot)
 
Retention?

Snapper and btbrk seem to be _the_ managers for btrfs.

**Comment by Torxed** [comment]
This is most likely coming, I think there's an issue on the topic :) https://github.com/archlinux/archinstall/issues/93
The goal is to have it for the next release (not the one that's dropping today/tomorrow) :)
